tests/TestHarness/testUtils.py

```diff
@@ -205,7 +205,8 @@ def checkDelayedOutput(popen, cmd, ignoreError=False):
         Utils.checkOutputFileWrite(start, cmd, output, error)
         if popen.returncode != 0 and not ignoreError:
             Utils.Print("ERROR: %s" % error)
-            raise subprocess.CalledProcessError(returncode=popen.returncode, cmd=cmd, output=error)
+            # for now just include both stderr and stdout in output, python 3.5+ supports both a stdout and stderr attributes
```

Comment:
I plan to address this comment in a follow-on PR. Plan on moving `stdout` to `output` and `stderr` to `stderr`. Then update all uses to use `stderr` instead of `output`; except for the one existing use that the `stdout` was added for.